packages/iframe-stamper/src/index.ts

```diff
@@ -132,9 +134,11 @@ export class IframeStamper {
 
     let iframe = window.document.createElement("iframe");
 
+    this.trustedParentOrigin = window.origin;
     // See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#sandbox
     // We do not need any other permission than running scripts for import/export/auth frames.
     iframe.setAttribute("sandbox", "allow-scripts allow-same-origin");
+    iframe.setAttribute("data-trusted-origin", this.trustedParentOrigin);
```

Comment:
is there a need to persist `window.origin` on the iframeStamper or can we do `iframe.setAttribute("data-trusted-origin", window.origin)` directly?

Reply:
Nope. Removed this in favor of the MessageChannel approach.